fastlane/actions/gitlab_create_project.rb

```diff
@@ -118,9 +118,16 @@ def self.migrate_snippets(gitlab_src, gitlab_dst, project_src, project_dst, user
         UI.message("Migrating snippets")
         snipptes = gitlab_src.snippets(project_src.id).auto_paginate.each do |snippet|
           code = gitlab_src.snippet_content(project_src.id, snippet.id)
+          UI.message("Snippet: '#{snippet.title}'")
+          if snippet.file_name.empty?
+            snippet_file_name = snippet.title.delete(' ') + "." + snippet.id.to_s
```

Comment:
'file_name' can contain only letters, digits, '_', '-', '@' and '.'
So here maybe a regex should be used to ensure this condition